packages/editor/src/editor/sidebar/fields/browser/useCmsBrowser.tsx

```diff
@@ -33,7 +33,7 @@ export const useCmsBrowser = (): Browser => {
     ),
 
     infoProvider: row => <CmsInfoProvider row={row} />,
-    applyModifier: row => ({ value: 'ivy.cms.co("' + (row.original.data as ContentObject).fullPath + '")' })
+    applyModifier: row => ({ value: "ivy.cms.co('" + (row.original.data as ContentObject).fullPath + "')" })
```

Comment:
why not use nice js template syntax? 
```suggestion
    applyModifier: row => ({ value: `ivy.cms.co('${(row.original.data as ContentObject).fullPath}')` })
```